cmd/podman/containers/rm.go

```diff
@@ -154,15 +152,9 @@ func setExitCode(err error) {
 	if registry.GetExitCode() == 1 {
 		return
 	}
-	cause := errors.Cause(err)
-	switch {
-	case cause == define.ErrNoSuchCtr:
+	if errors.Is(err, define.ErrNoSuchCtr) {
 		registry.SetExitCode(1)
-	case strings.Contains(cause.Error(), define.ErrNoSuchCtr.Error()):
```

Comment:
There is a reason for the string compare. We cannot keep the same error over the http API on podman-remote.